### DISCUSSION TITLE
Idealistic Anarco-collectivist ecosystem needs help Idealists and becomers needed. If you build it they will come!

### DISCUSSION THREAD

**Discussion by AnarchSun** [Cyfrin/foundry-full-course-cu#2161] [Ideas]
I am learning developpment on my own nand i love it. I ncan see i high potential in blockchain ntechnologies. Being an anarchist, I dream of changing the world and politics using blockchain technology. I need dreamer developpers with free time and conviction in order to chanmg governments. I dream nof building a blockchain and ecosystem for Québec Politics in norder the dismatel state and distribute the bpower to nthe people. Smart contract voting and propaganda network. Québec has tons of hydroelectricity and water... Water and power hahah! I would build a node validator network like pi network that would allow people to mine from a smartphone and computer backed with hydro... I like vchain ans Solana cause they are cheap secure and fast now lets see what is possible with zkSync... I lkike those chaine because they are not to mainstream and they are alternatives to etherium wich is very expensive and limited on growth. The project need a network like pi using water that evaporates and pollutes with time used for liking leaving place for new ideas to emmerge to nthe top. It would be used to teach inform and consult the people plus voting. Instead of having stupid ripoff game adds ads silly facebook paying games and candy crush losses of time to give them nto watch for tokens to post, it would be iinformation adds and propaganda. The network would pay people to nparticipate in decisions and learn nwatching goverment stuff (the goal is to dismantel state and give all the power to the people.. . Need an Nft market for culture etc whatever is needed to run collective interests. Paying people to run their own politics like coinbase learning activities that give coins. I also like the vchain holding mechanism that yeilds vthor just for holding vet in wallet... same principles but anarchist way. Anarchism is very anti capitalist because money corrupts... This is our chance to use blockchain in our advantage for freedom. Consulting people at every important decision not only one every 4 years... Thats bullshit!. build a wallet for politics... social insurance number gives access to community decisions and political action. Sky is not the limit. Lets take the power! And give it back! Expropriation if needed... Just kidding... Become don't Beleive! All i want is the privilege of organising and namign stuff. Forget about the money It should help the community not ego.. But building such a project makes it easy to earn a better living in the long run... Being there first in crypto means a lot because of growth...

**Comment by EngrPips** [comment]
Hello @RastaDjuss, Well Said. I Look forward to contributing to web3 immensely myself

**Reply by AnarchSun** [reply]
Nice feel free to invite sponsors and extra utopists beleiving in nthe potential of the project and its Anarco-Collectivist values. I created a Git repository for collaboration... I am new at this. I'm not quite sure how and where to start and wich way i should go i'm just 15-20 videos into the Cyfrin Course. Just deployed my first favorite number smart contract on the sepolia testnet... Thats it. Then i've been fooling around with coding through Generating and publishing Large Nft Collections Punks-not-dead [(Punks-not-dead)](https://opensea.io/punks-not-dead-8) on Etherium and another 10k collection wich is a "BiT" of a Joke because its all Bits (penises=influence) Here vit is:[Bit](https://opensea.io/collection/nit-69)... I made itb through that dilema on my own knowing nothing about how to do it. Lots of trial and fucking node version error distractions and patience. Then I started playing around with repositories and even doccker images and containers etc... With docker desktop and docker hub.  Just mostly folling around with installation instuctions in a maze on branches... I deployed app models  and tyhe solana coockbook... I am the one and only yarn install master!

**Comment by AnarchSun** [comment]
great pleased to meet you… Tell me more about yourself im curious. and welcome.. Feel free to recrute people that share the same utopist interests and cause. This is my git repo page… I had a new repo started onn gitkracken desktop nthinking it would link to github but i guess not i had a readme started and 2 repos on blockchain developement in there… Just started it yesterday: https://github.com/RastaDjuss in there theres mostly solana stuff and being a beginner and self tought i lack some knoledge in order to create this network and ecosystem… I need people that know about the different blockchains to see witch one suits best goverment networks and voting plus community etc… I thought starting from scratch could be interesting but i think that being decentraliuzed, it doesn’t really matter who started the blockchain and what they do that much when voting is implicated. One thing is cerrtain is it must follow anarchist morals and ecolgy etc principles...

**Reply by AnarchSun** [reply]
I forgot the linkmto the project repository: [AnarCrypt-EcO-SubSystem](https://github.com/RastaDjuss/Anarcrypt-eco-subsystem.git) but i want to start an norganisation ton facilitate collaboration... I'm not sure what to do I'll fly off to my Blockchqain and smart contract solidify course... Function exit!

**Reply by EngrPips** [reply]
I wish you the very best in your pursuit, Friend.
